lib/types.ts

```diff
@@ -36,6 +36,32 @@ export interface Link {
   }
 }
 
+export interface Team {
+  sys: {
+    space?: Link
+    status?: Link
```

Comment:
Is the `status` really an entity?

Reply:
status is not an entity and it's not in `sys` it's a top-level field outside of the `sys` object

Reply:
Sounds reasonable. I just copied the type from cma ([Source](https://github.com/contentful/contentful-management.js/blob/master/lib/entities/team.ts#L7)):
```
export type TeamProps = {
  sys: MetaSysProps & { memberCount: number; organization: { sys: MetaLinkProps } }
  name: string
  description: string
}
```
The [MetaSysProps defines the status as a link](https://github.com/contentful/contentful-management.js/blob/master/lib/common-types.ts#L141). I updated this type but do one of you know why the cma is typing status as a link?